Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -246,7 +246,7 @@ final class PackagePIFProjectBuilder: PIFProjectBuilder {
         super.init()
 
         guid = package.pifProjectGUID
-        name = package.manifestName // TODO: use identity instead?
+        name = package.manifest.displayName // TODO: use identity instead?
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
Same as above.